docs/reference/dune/README.md

```diff
@@ -0,0 +1,42 @@
+| [veBAL](https://dune.com/balancer/vebal)                   | Total veBAL, veBAL locking, power by top LPs, votes by gauge and network                                            |                                          |
+| [veBAL Analysis](https://dune.com/balancer/vebal-analysis)          | Power and votes by provider                                                                                    | Provider                                 |
+| [veBAL Gauge Votes](https://dune.com/balancer/vebal-gauge-analysis)    | Voters on gauge                                                                                                 | Gauge                                    |
+| [Balancer Overview](https://dune.com/balancer/balancer-overview)       | Volume (by source, cumulative, 24h and 7d), TVL, average swap fees, liquidity utilization, Balancer on 1inch and cowswap | Pool ID, Start Date, End Date, and blockchain |
+| [Balancer Exchange](https://dune.com/balancer/balancer-exchange)       | Volume (by token, 24h and 7d), # of swaps, new/old traders, fees distribution, gas costs                           | Aggregation, Start Date, End Date, and blockchain |
+| [Balancer Report](https://dune.com/balancer/balancer-report)        | TVL, Volume (cumulative, market share, by token), swap fees revenue                                                | Pool ID, Start Date, End Date, and blockchain |
+| [Volume](https://dune.com/balancer/volume)                      | 24h, 7d, 30d, moving averages, volume by pool, daily volume changes, by blockchain (median swap, # of swaps, new traders, Net token inflow/outflow |                                          |
+| [Balancer Volume Breakdown](https://dune.com/balancer/balancer-volume-breakdown) | Overview on volume by token, pool, and source                                                                  | Source Address, -Source Name, Pool ID, Start Date, End Date, and blockchain |
+| [Balancer Volume - Source Breakdown](https://dune.com/balancer/Balancer-Volume-Source-Breakdown) | Weekly, daily, and hourly volume breakdown by source (DEXs and Aggregators, Heavy Traders and MEV bots). Addresses used are listed on this [link](https://dune.com/queries/3004790) | Source Address, -Source Name, Pool ID, Start Date, End Date, and blockchain |
+| [Balancer Volume - Pool Breakdown](https://dune.com/balancer/Balancer-Volume-Pool-Breakdown)   | Weekly, daily, and hourly volume breakdown by top pools                                                         | Start Date, End Date, and blockchain      |
+| [Balancer Volume - Token Breakdown](https://dune.com/balancer/Balancer-Volume-Token-Breakdown) | All-time volume by token, Monthly and hourly top tokens volume                                                    | Blockchain, Aggregation, Top x tokens    |
```

Comment:
could remove `Balancer` from the beginning of each dashboard name, also reminds me we could edit the URLs - for instance, from `balancer/Balancer-Volume-Pool-Breakdown` to `balancer/volume-pool-breakdown`

Reply:
good idea! fixed on docs/will fix on dune